AnnoyancesFilter/Widgets/sections/widgets.txt

```diff
@@ -169,6 +169,9 @@ blog.livedoor.jp##div[id^="shitarss_title_"]
 !
 ! NOTE: Blogroll specific rules
 !
+sensualgirls.org##.cbox2_cont.center
+||sensualgirls.org^$script
```

Comment:
I think that it might not works in this case, because there is `/flr7.js`.
If I'm not wrong, it could be also added to the scriptlet `#%#//scriptlet("abort-current-inline-script", "document.dispatchEvent", "/getexoloader/")`:
https://github.com/AdguardTeam/AdguardFilters/blob/58daeaf2e204f6075cb92bc8d375834f41d07746/BaseFilter/sections/general_extensions.txt#L25

Reply:
@Alex-302 if you mean /flr.js$domain=
It is not working

Reply:
@AdamWr so as yours